### ISSUE TITLE
Can't FileConverter 1.2.3 or 2.0.2 to run on Windows 10 x64

### ISSUE THREAD

**Issue by tted3286** [Tichau/FileConverter#444]
Can't FileConverter 1.2.3 or 2.0.2 to run on Windows 10 x64

You specify on your site that visual studio 2022 community should be installed, but not which individual components; ideally I install almost nothing.  A base install is about 1.4 GB.  Which individual components should be selected?  

[url=https://imgbb.com/][img]https://i.ibb.co/0nGGV9Q/error.jpg[/img][/url]
[Diagnostic.zip](https://github.com/Tichau/FileConverter/files/15228624/Diagnostic.zip)

**Comment by tted3286** [comment]
On another system I get this error before installing visual studio community 2022, after installing it nothing changes.

[OtherSystemDiagnostic.zip](https://github.com/Tichau/FileConverter/files/15228713/OtherSystemDiagnostic.zip)

**Comment by tted3286** [comment]
Fine I solved it myself:
FileConverter 2.0.2 Requires Windows 1903 and up, and of course, all of .net kept which .net 4.8+.
As for the middleware, most have no system requirements:
-> ffmpeg; no system requirements (videos)
-> ImageMagick; no system requirements (images)
-> ghostscript; no system requirements (pdf)
-> Sharpshell;  C# which is a part of the .net 6 frameworks which a common software prerequisite
-> Audio conversion; no system requirements

-> AnimatedGifs; might require Visual Studio 2022 base install with WPF and XAML components.

In summary unless you convert AnimatedGifs, FileConverter 2.0.2 requires Windows 10 1903 or higher and .net 6.0 or higher (install both x86 and X64).  If you have removed anything from .net with NTLite, it will not work unless you keep the whole thing.  .Net 3.5 is not required nor used.  Obviously anything relating to xaml and WPF in NTlite should also be kept.  Rant Over.  I have fixed all users problems except nobody will know to look here.
